pkgs/applications/altcoins/zcash/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, libsodium, fetchFromGitHub, wget, pkgconfig, autoreconfHook, openssl, db62, boost
+, zlib, gtest, gmock, miniupnpc, callPackage, gmp, qt4, utillinux, protobuf, qrencode, libevent
+, withGui }:
+
+let snark        = callPackage ./snark { inherit boost openssl; };
```

Comment:
If I remember correctly the snark package has a bunch of zcash specific stuff so it's probably not useful outside of this package atm...